language/move-core/types/src/vm_status.rs

```diff
@@ -536,6 +702,21 @@ impl StatusCode {
 
         StatusType::Unknown
     }
+
+    /// Should only be used in debug asserts. These status codes are missing some data
+    /// checked in debug asserts
+    pub fn should_skip_checks_for_todo(self) -> bool {
```

Comment:
@dariorussi I don't know how to set this up properly. But some checks don't have their bytecode offest/`pc` set during execution. No time to figure this out just now, so I need this todo check splattered in front of some debug asserts.

Reply:
it looks all beautiful to me....